webapp/components/create_comment.jsx

```diff
@@ -263,6 +263,25 @@ export default class CreateComment extends React.Component {
         $('.post-right__scroll').parent().scrollTop($('.post-right__scroll')[0].scrollHeight);
 
         this.setState({message});
+
+        this.checkMessageLength(message);
+    }
+
+    checkMessageLength(message) {
```

Comment:
great suggestion @enahum, let me take a look at how we might be able to accomplish that. Maybe by moving most of the code to the shared `Textbox` component...